tick/CMakeLists.txt

```diff
@@ -36,14 +36,28 @@ if (${GTEST_FOUND})
     enable_testing()
     message(STATUS "Testing in C++ enabled")
 
-    find_package (Threads REQUIRED)
+    find_package(Threads REQUIRED)
+    find_package(PythonInterp REQUIRED)
+    find_package(PythonLibs REQUIRED)
 
     include_directories(${GTEST_INCLUDE_DIRS})
 
+    set(TICK_TEST_LIBS
+        ${PYTHON_LIBRARIES}
+        ${GTEST_BOTH_LIBRARIES}
+        ${CMAKE_THREAD_LIBS_INIT})
+
     if(APPLE)
```

Comment:
No, docker does not play in here. It's only because OSX handles rpaths a bit different than linux